app/TuistApp/Sources/Views/DevicesView/DevicesViewModel.swift

```diff
@@ -56,22 +56,43 @@ struct PinnedSimulatorsKey: AppStorageKey {
 
 struct SelectedDeviceKey: AppStorageKey {
     static let key = "selectedSimulator"
-    static let defaultValue: SelectedDevice? = nil
+    static let defaultValue: ReducedDevice? = nil
 }
 
 enum SelectedDevice: Codable, Equatable {
```

Comment:
```suggestion
enum Device: Codable, Equatable {
```

This doesn't necessarily represent a selected device only